### PR TITLE
Mention HA in multinode section.

### PR DESCRIPTION
**Pull request by alejandrodnm**
# Description

While reading the docs about multi node, I ended up with the feeling
that you could only have a single access node and that seemed like a
single point of failure. I had to keep reviewing the docs to find the HA
section for multi node.

I think a breif mention of HA for multinode in the setup page will make
readers have a better understanding of their options, and that there's a
difference between the two things.

cc @jfjoly 

# Links

Fixes #[insert issue link, if any]

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-setup.md [hunk 1]
```diff
@@ -66,8 +68,9 @@ and architecture, see the
 When you have set up your multi-node installation, you can configure your
 cluster. For more information, see the [configuration section][configuration].
 
-[configuration]: /how-to-guides/multinode-timescaledb/multinode-config/
-[install]: /install/latest/
-[setup]: /install/latest/
-[about-multi-node]: /how-to-guides/multinode-timescaledb/about-multinode/
+[configuration]: /timescaledb/:currentVersion:/how-to-guides/multinode-timescaledb/multinode-config/
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply timescaledb/how-to-guides/multinode-timescaledb/multinode-setup.md:71]: Thanks, we just changed how that link checker works and we're still going through and finding things. Glad you worked it out 💖

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-setup.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ To set up multi-node on a self-hosted TimescaleDB instance, you need:
 The access and data nodes must begin as individual TimescaleDB instances.
 They should be hosts with a running PostgreSQL server and a loaded TimescaleDB
 extension. For more information about installing self-hosted TimescaleDB
-instances, see the [installation instructions][install].
+instances, see the [installation instructions][install]. Additionally, [high 
+availability with multi-node][multi-node-ha] can be configured to increase 
+redundancy and resilience.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/multinode-timescaledb/multinode-setup.md:20]:
```suggestion
instances, see the [installation instructions][install]. Additionally, you
can configure [high availability with multi-node][multi-node-ha] to 
increase redundancy and resilience.
```

Small reword to make it more active, and also get the link on one line.

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-setup.md (hunk 2 above)
Comment (alejandrodnm) [reply timescaledb/how-to-guides/multinode-timescaledb/multinode-setup.md:20]: I love working with people that take my bad writing and change it into somethings awesome.

**Comment by jfjoly** [comment]
LGTM, thanks for adding this !

**Review by jfjoly — APPROVED** [review]

**Comment by alejandrodnm** [comment]
Hi @Loquacity this just needs your approval to be merged :)

**Review by Loquacity — APPROVED** [review]

**Pull request merged**